thinc/api.py

```diff
@@ -2,7 +2,7 @@
 from .initializers import normal_init, uniform_init, xavier_uniform_init, zero_init
 from .loss import categorical_crossentropy, L1_distance, cosine_distance
 from .loss import sequence_categorical_crossentropy
-from .model import create_init, Model, serialize_attr, deserialize_attr
+from .model import CreateInit, Model, serialize_attr, deserialize_attr
```

Comment:
```suggestion
from .model import create_init, Model, serialize_attr, deserialize_attr
```
I think it's better to keep spelling this lower-cased. We have plenty of functions that are camel cased that return `Model`, after all.